aries-site/src/examples/templates/forms/do-dont/ColumnFormDont.js

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+import { Box, Button, ResponsiveContext } from 'grommet';
+import { ContactInformation, ShippingInfomation } from '../Shipping';
+import { ShippingExample } from '../ShippingExample';
+
+export const ColumnFormDont = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <ShippingExample>
+      <Box gap="large" direction="row">
+        <ShippingInfomation />
+        <Box>
+          <ContactInformation />
+          <Box
+            align={!['xsmall', 'small'].includes(size) ? 'start' : undefined}
```

Comment:
Can we write this as what we want, as opposed to what we don't want?

```suggestion
            align={['xsmall', 'small'].includes(size) ? undefined : 'start'}
```